### PR TITLE
Long press on images show Private Mode icon option and translation #786

### LINKED ISSUE #786: Long press on images show Private Mode icon option and translation
IMG_6092CE263CBC-1

instead of private mode we should refer to "Forget Mode"

On this occation please take a look which other part of code uses Private Mode translation and assets and remove/replace them all.

### PR DESCRIPTION
**Pull request by shahenAnt**
Fix #786 

## Checklist:

- [ ] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [ ] I updated or created necessary unit tests
- [ ] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift [hunk 1]
```diff
@@ -140,7 +140,7 @@ extension BrowserViewController: WKUIDelegate {
                 })
             }
 
-            actions.append(UIAction(title: Strings.ContextMenuOpenInNewPrivateTab, image: UIImage.templateImageNamed("menu-NewPrivateTab"), identifier: UIAction.Identifier("linkContextMenu.openInNewPrivateTab")) { _ in
+            actions.append(UIAction(title: Strings.ContextMenuOpenInNewPrivateTab, image: UIImage.templateImageNamed("menu-ForgotMode"), identifier: UIAction.Identifier("linkContextMenu.openInNewPrivateTab")) { _ in
```
Comment (chrmod) [inline Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift]: it should probably be called `ForgetMode`

File: Client/Frontend/Menu/Menu.xcassets/menu-ForgotMode.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (chrmod) [inline Client/Frontend/Menu/Menu.xcassets/menu-ForgotMode.imageset/Contents.json]: Don't we have this asset already? Would be beter to use same icon

File: Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift (hunk 1 above)
Comment (shahenAnt) [reply Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift]: Done

File: Client/Frontend/Menu/Menu.xcassets/menu-ForgotMode.imageset/Contents.json (hunk 2 above)
Comment (shahenAnt) [reply Client/Frontend/Menu/Menu.xcassets/menu-ForgotMode.imageset/Contents.json]: Done

**Pull request merged**